src/ca_roots.jl

```diff
@@ -74,7 +74,7 @@ const BSD_CA_ROOTS = [
 ]
 
 const SYSTEM_CA_ROOTS_LOCK = ReentrantLock()
-const SYSTEM_CA_ROOTS = Ref{String}()
+const SYSTEM_CA_ROOTS = Ref{Union{String,Nothing}}(nothing)
```

Comment:
```suggestion
const SYSTEM_CA_ROOTS = Ref{Union{String, Nothing}}(nothing)
```
The pickiest of nits. Just because the other one has the space.